src/pyOpenMS/pyopenms/plotting.py

```diff
@@ -220,33 +218,34 @@ def plot_chromatogram(c: "MSChromatogram"):
 def _annotate_ion(mz: float, intensity: float, annotation: Optional[str],
                   color_ions: bool, annotate_ions: bool, matched: Optional[bool],
                   annotation_kws: Dict[str, object], ax) -> Tuple[str, int]:
+    """Annotate a specific fragment peak.
+
+    :param mz: The peak's m/z value (position of the annotation on the x axis).
+    :type mz: float
+
+    :param intensity: The peak's intensity (position of the annotation on the y axis).
+    :type intensity: float
+
+    :param annotation: The annotation that will be plotted.
+    :type annotation: str, optional
```

Comment:
The sphinx docs are not very clear about this optional, but it looks correct. We need to check how this renders.